src/Routers/FeaturesRouter.js

```diff
@@ -20,10 +20,10 @@ export class FeaturesRouter extends PromiseRouter {
     if (!req.auth.isMaster) {
       return masterKeyRequiredResponse();
     }
-    
+
     return Promise.resolve({
       response: {
-        results: [getFeatures()]
+        results: getFeatures()
```

Comment:
Eh, if we are not going to follow the standard exactly I would rather go back to just using a raw object with no nested layer.